app/src/main/java/tech/pauly/findapet/data/SexConverter.java

```diff
@@ -0,0 +1,25 @@
+package tech.pauly.findapet.data;
+
+import android.arch.persistence.room.TypeConverter;
+
+import tech.pauly.findapet.data.models.Sex;
+
+public class SexConverter {
+
+    @TypeConverter
+    public static Sex toSex(int code) {
```

Comment:
When adding the next filter, just put these in the enum itself, don't like it being separate.